Functions/BIDS/xASL_bids_MergeNifti.m

```diff
@@ -214,6 +217,20 @@
 		return;
 	end
 	
+	% Set priorities for merging JSONs, higher number = higher priority
```

Comment:
This is a bit tricky perhaps.

1) Note that in the case of GE, it is a deltaM image, not CBF (which it spits out separately). So deltaM should probably go before m0scan. And control/label as well?
2) I don't think we should generalize this. For BIDS, we want to keep NIfTIs as how they were acquired. GE/dcm2niiX just has a bug by splitting their single sequence here. Siemens as well I guess. But the generalized solution is to keep the output as it was.

Reply:
1. deltaM HAS the highest priority - higher number has the highest priority (as said in the comment in the code). so deltaM overwrites all. deltaM > control/label > m0scan > cbf (this is there just for completeness). I don't understand your comment, deltaM already goes before anything else.

2. Indeed, we are not generalizing. All this priority stuff is done in a GE subfunction only applying for GE. For Siemens and Philips - JSONs are merged with equal priorities.

Reply:
1. Ah I see now. Then I would just mention them in the order such that 1 and 0 are together, now this is confusing to read. So keep the code as is, just 4 3 2 1 0 instead of 1 2 3 4. If that's done, you can resolve this comment.
2. OK, this prioritization of merging could actually be general. Also, since you reuse it in xASL_bids..._Merge, this sounds generic. But fine for now how it is.

Reply:
I've clarified this in the comment. I don't agree to merge 0 and 1 because a known file type (CBF) should have a higher priority than unknown (that has a zero priority).

Changing the order of code doesn't make any sense - it won't affect the readability at all. Sorry, but not doing things that don't make any sense ;) The options are listed in order of increasing priority. Otherwise is always listed at the end.

Just as an FYI: The prioritization function is general. But the setting of priorities is only used in specific cases when we know.